locale/ar/about/index.md

```diff
@@ -38,7 +38,7 @@ server.listen(port, hostname, () => {
  تأخذ النود جي اس موديل الاحداث (event model) ابعد قليلا فتمثل الحلقة التكرارية الخاصة بالاحداث ([event loop][]) كمكون اساسي في وقت التشغيل (runtime construct) وليس كمكتبة في انظمة أخرى، حيث ان هنالك دائما استدعاء متزامن (blocking call) 
 للبدء في حلقة الاحداث.
 مبدئيا، يتم تحديد السلوك عبر دالة مستدعاة في بداية السكريبت في نهايتها 
-تقوم بتشغيل خادم (server) عبر استدعاءٍ غير متزامن (blocking call) مثل <span dir="ltr">`EventMachine::run()`</span>، ولكن في النود جي اس لا يوجد شيء من هذا القبيل.
+تقوم بتشغيل خادم (server) عبر استدعاءٍ غير متزامن (non-blocking call) مثل <span dir="ltr">`EventMachine::run()`</span>، ولكن في النود جي اس لا يوجد شيء من هذا القبيل.
```

Comment:
`blocking` is correct here, see https://nodejs.org/en/about/:

> Typically behavior is defined through callbacks at the beginning of a script and at the end starts a server through a **blocking** call like `EventMachine::run()`. In Node there is no such start-the-event-loop call.

Reply:
ok, Corrected the mistake.
@fhemberger